### PR TITLE
Gsoc19

### PR DESCRIPTION
**Pull request by xadahiya**
Fixed typos in #12

### CODE REVIEW

**Comment by xadahiya** [comment]
We'll probably need to add a couple of things like proposal template etc. Will try to finish this tonight.

File: pages/GSoC-19.md [hunk 1]
```diff
@@ -1,8 +1,153 @@
...
-# Ideas will go here
\ No newline at end of file
+---
+layout: page
+title: Ideas page for GSoC 2019 | Hydraecosystem.org
+permalink: /gsoc-19
+---
+
+# Introduction
+Welcome to Python HYDRA ideas page for [GSoC 2019](https://summerofcode.withgoogle.com/). We would like to receive proposals that encompass different aspects of the development. Different tools are under development
```
Comment (Mec-iS) [inline pages/GSoC-19.md]: change name from Python Hydra

Comment (xadahiya) [reply pages/GSoC-19.md]: Hydra ecosystem?

File: pages/GSoC-19.md [hunk 2]
```diff
@@ -1,8 +1,153 @@
...
+- ## Proposal template
+[Here](https://docs.google.com/document/d/1qPR02o6jY4uFBdCf3S0ous4LG32YCe6ZJuDGojXH2JY/edit)'s a proposal template for you to get started.
+
+## Communication Channels
+- [Gitter](https://gitter.im/HTTP-APIs/Lobby)
+- [W3C Group](https://www.w3.org/community/hydra/)
+- [Slack](https://httpapis.slack.com)
+- [Mailing List](public-hydra@w3.org)
+- Email
+    - [Lorenzo Moriondo](mailto:tunedconsulting@gmail.com)
```
Comment (Mec-iS) [inline pages/GSoC-19.md]: scramble emails?

File: pages/GSoC-19.md [hunk 3]
```diff
@@ -1,8 +1,153 @@
...
+    - hydrus
+    - [Graph Algorithms](https://en.wikipedia.org/wiki/Category:Graph_algorithms)
+
+    **Difficulty Level** : *Intermediate/Hard*
+    
+    **Issues**<br/>
+    Some issues are already open for this idea, please [check this list](https://github.com/HTTP-APIs/python-hydra-agent/issues?q=is%3Aissue+is%3Aopen+label%3AGSOC-2019):
+
+- ## General Improvements in hydrus.
+**Main Idea**:<br/>
```
Comment (Mec-iS) [inline pages/GSoC-19.md]: titles and subtitles have to use the heading markdown: `##`

File: pages/GSoC-19.md [hunk 4]
```diff
@@ -1,8 +1,153 @@
...
+There is always room from improvement in our flagship server, hydrus:
+    - implementation of a search mechanism (use of `hydra:search` and IRITmeplates, and possibly Triple Pattern Fragments)
```
Comment (Mec-iS) [inline pages/GSoC-19.md]: typo in "IRITemplates"

File: pages/GSoC-19.md (hunk 1 above)
Comment (Mec-iS) [reply pages/GSoC-19.md]: yep

**Comment by Mec-iS** [comment]
we have also to change the header of the website from Python Hydra

File: pages/GSoC-19.md (hunk 2 above)
Comment (xadahiya) [reply pages/GSoC-19.md]: Not sure how to do that 😅

Comment (Mec-iS) [reply pages/GSoC-19.md]: like `username [AT] domain [DOT] com` to avoid emails scrapers

**Comment by xadahiya** [comment]
@Mec-iS done

**Comment by xadahiya** [comment]
We need some logo. Do you have anything in mind?

**Comment by Mec-iS** [comment]
i am thinking about it. For now just remove the logo and change the name in the header please (in a new PR)

**Comment by xadahiya** [comment]
@Mec-iS #14

**Pull request merged**